darwin/DarwinMachine.c

```diff
@@ -47,7 +47,7 @@ static void DarwinMachine_freeCPULoadInfo(processor_cpu_load_info_t* p) {
    *p = NULL;
 }
 
-static size_t DarwinMachine_allocateCPULoadInfo(processor_cpu_load_info_t* p) {
+static unsigned int DarwinMachine_allocateCPULoadInfo(processor_cpu_load_info_t* p) {
```

Comment:
Any particular reason for changing the signature here to `unsigned int`? Too many callers?

Potentially update `activeCPUs`/`existingCPUs` to `size_t`?